### ISSUE TITLE
[Feature] - Hud messages cache

### ISSUE THREAD

**Issue by Dockter** [AlmuraDev/Almura#532]
We send popup notifications to the client, it would be neat if the client cached them and then a GUI could be brought up of previously sent messages during that logon session.

**Comment by Grinch** [comment]
Is this something we still want to do? It shouldn't be too hard to implement a notification center that stores the current client/server session notifications.

**Comment by Dockter** [comment]
Yes, I would still like this.  Several times I've missed messages and I would like to be able to see them.  It should survive the server connection but not the client closing.
